packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -38,6 +38,11 @@ export const concepts = [
         title: "How Okta works",
         path: "/docs/concepts/how-okta-works/"
       },
+      {
+        title: "Identity engine",
+        path: "/docs/concepts/ie-intro/",
+        hidden: true
```

Comment:
are we hiding this concept @jakubvul-okta @emekaafigbo-okta ? One big reason for the concept is so we can link OIE guides and references to this concept. Does it make sense to keep it hidden from the nav?